File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/member/ImageExtension.java
package com.example.cf_sdk.logic.changebanksdk.model.member;

import com.google.gson.annotations.SerializedName;

/**
 *
 * Enum for the Image Extension.
 */

public enum ImageExtension {

    @SerializedName("PDF")
    PDF("pdf"),

    @SerializedName("PNG")
    PNG("png"),

    @SerializedName("JPEG")
    JPEG("jpeg");

    String mExtensionName;

    ImageExtension(String extensionName) {
        mExtensionName = extensionName;
    }

    public String getExtensionName() {
        return mExtensionName;
    }

}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/member/ImageExtension.kt
package com.example.cf_sdk.changebankapi.model.member

import com.google.gson.annotations.SerializedName

/**
 *
 * Enum for the Image Extension.
 */
enum class ImageExtension(extensionName: String) {
    @SerializedName("PDF")
    PDF("pdf"), @SerializedName("PNG")
    PNG("png"), @SerializedName("JPEG")
    JPEG("jpeg");

    var extensionName = ""

    init {
        this.extensionName = extensionName
    }
}